vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/LocatorType.java

```diff
@@ -26,6 +26,11 @@
 
     String getKey();
 
+    default Locator locatorFactory(String searchValue)
+    {
+        return new Locator(this, searchValue);
```

Comment:
this way we introduce cyclic dependency, but let's keep it as is in this PR, and change approach in another commit later